### PR TITLE
Bug/jenkins 37603 favorite hover state

### PR DESCRIPTION
**Pull request by cliffmeyers**
# Description
- The bulk of the change was to the favorite code in the JDL reviewed in jenkinsci/jenkins-design-language#88
- See [JENKINS-37603](https://issues.jenkins-ci.org/browse/JENKINS-37603).
- See [JENKINS-36895](https://issues.jenkins-ci.org/browse/JENKINS-36895).

# Submitter checklist
- [x] Link to JIRA ticket in description, if appropriate.
- [x] Change is code complete and matches issue description
- [x] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [x] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [x] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
   - [ ] Check hover states of Favorite stars on Pipelines, Pipeline Detail's Activity and Branches tabs
   - [ ] Check alignment of the favorite stars in the main pipelines table - ensure the table columns are still appropriately sized.
   - [ ] Increase / decrease zoom level of page and verify that favorite stars are not clipped in Pipelines list nor in Favorite cards at top.
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@jenkinsci/code-reviewers @reviewbybees

### CODE REVIEW

**Comment by cliffmeyers** [comment]
ATH passed locally.

`2016-09-07 15:21:17,962 [Thread-3] INFO   (InputStreamHandler.java:47) com.github.eirslett.maven.plugins.frontend.lib.DefaultGulpRunner - OK. 330  total assertions passed. (4m 15s)`

File: blueocean-dashboard/package.json [hunk 1]
```diff
@@ -1,71 +1,71 @@
 {
-  "name": "blueocean-dashboard",
```
Comment (kzantow) [inline blueocean-dashboard/package.json]: :( for unnecessary full-file change?

**Comment by kzantow** [comment]
:ant: a bunch of unrelated changes, other change looks :bee:

File: blueocean-dashboard/package.json (hunk 1 above)
Comment (cliffmeyers) [inline blueocean-dashboard/package.json]: These package.json files use an indenting inconsistent with the rest of the project.  I feel like it should be fixed eventually so my editor isn't using the wrong indent every time I make a change to the files. No good? :(  Or should it just be done in a separate PR?

Comment (kzantow) [inline blueocean-dashboard/package.json]: I mean, it's fine, just makes the PR confusing.

**Comment by i386** [comment]
@cliffmeyers great job 💯
